earth_enterprise/rpms/build.gradle

```diff
@@ -251,20 +254,54 @@ task openGeeCommonDeb (type: Deb) {
 task openGeeServerRpm (type: Rpm) {
     requires('opengee-common', openGeeVersion, GREATER | EQUAL)
     packageName = 'opengee-server'
+    release = '1'
```

Comment:
RPM packages are versioned by package release, as well as by project/source version, as packaging is often maintained separately from the project source tree, and both version and release are used when computing updates.  I am not presuming its always going to be 1, just that it will always get reset back to 1 with each new source version.

Reply:
So, is it Gradle that is responsible for incrementing this?

Reply:
This is not an action for this PR, but FYI we were thinking about RPM versions in the 5.2.0 release, so the git tags for the 5.2.0 release are valid RPM versions.  For example, the final version of 5.2.0 is actually tagged `5.2.0-3.final` so that it comes after the pre-releases in RPM versioning.  We may want to incorporate that into our RPM building process in the future.